Update home page content

rmzelle opened the pull request:
@adam3smith, can you review?

rmzelle committed:
Update home page content

adam3smith commented on _pages/home.md:
```diff
@@ -87,7 +87,14 @@ multicolumn_list:
+This website is divided into several sections:
 
-Our crowdsourced repository offers over 8000 free CSL citation styles, and CSL is used by commercial and open source products alike, including:
+* [Authors](/authors/) - learn how to obtain and use our CSL citation styles
```
how about "find and use" -- obtain sounds odd to me.

adam3smith replied:
Would have to look, but might work better to also bold the category. 
Also, I think these should be endashes (`&endash;`) not hyphens.

adam3smith commented on _pages/home.md:
```diff
@@ -87,7 +87,14 @@ multicolumn_list:
 ---
 
-Visit our [Authors](/authors/) section to learn how to obtain and use our styles; [Publishers](/publishers/) if you are a publisher or journal editor with an interest in CSL; [Developers](/developers/) if you want to edit or create styles, are looking to support CSL in your own software, or wish to contribute to development of CSL itself; [Blog](/blog/) for big news updates; [About us](/contact/) ...
+This website is divided into several sections:
```
Do we need this? It sounds so formal. Wouldn't just the list do?

adam3smith reviewed (commented):
Looks good. A couple of suggestions

rmzelle committed:
Hide title on home page

Also bold “Citation Style Language” and keep it on a single line
Change site title

rmzelle commented on _includes/page__hero.html:
```diff
@@ -27,13 +27,6 @@
-        {% else %}
-          {{ page.title | default: site.title | markdownify | remove: "<p>" | remove: "</p>" }}
-        {% endif %}
-      </h1>
```
@pmelchor, do you know from the top of your head if it's okay to change this template file? I want to suppress the page title on the homepage.

pmelchor replied:
As a best practice you want to keep a visible and unique `<h1>` main heading on each page. This is relevant for SEO and other markup related purposes.

If you don't like the design we can tweak that and/or find a different placing for the `<h1>` on the homepage.

In terms of the template, I think that right now we are only using that one for the homepage.

rmzelle replied:
Okay, I'll put it back :).

rmzelle committed:
Implement Sebastian's suggestions

rmzelle committed:
Tighten up home page a bit

rmzelle committed:
Tighten up intro

rmzelle committed:
Eliminate a few more words

rmzelle commented:
> Looks good. A couple of suggestions

Thanks! Made a few more edits, and done for now. Feel free to merge if you're happy with this.

rmzelle commented:
(and squash please)

rmzelle committed:
Restore title on home page

rmzelle committed:
Reduce text size of multicolumn list items

rmzelle committed:
Merge branch 'master' into home-page

rmzelle committed:
Simplify CSS
